app/models/in_person_enrollment.rb

```diff
@@ -70,6 +71,10 @@ def on_status_updated
     self.status_updated_at = Time.zone.now
   end
 
+  def set_unique_id
+    self.unique_id = InPersonEnrollment.generate_unique_id
```

Comment:
Since this is a method in the same class, do we need to reference the class?

```suggestion
    self.unique_id = generate_unique_id
```

Reply:
Ruby doesn't scope static/class methods to instances like that. If you wanted to remove a hardcoded class ref you could do:
```suggestion
    self.unique_id = self.class.generate_unique_id
```

Reply:
Ah, I missed that the other method was a class method.

Reply:
Updated to `self.class` in 5a8f17baf